client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useAllDepositsOverviews } from 'wcpay/data';
+import { Flex, TabPanel } from '@wordpress/components';
+
+/**
+ * Internal dependencies.
+ */
+import { fundLabelStrings } from './strings';
+import { getCurrencyTabTitle } from './utils';
+import BalanceBlock from './balance-block';
+
+/**
+ * Renders an account balances panel with tab navigation for each deposit currency.
+ *
+ * @return {React.FunctionComponent} Rendered balances panel with tab navigation for each currency.
+ */
+const AccountBalancesTabPanel: React.FC = () => {
+	const {
+		overviews,
+		isLoading,
+	} = useAllDepositsOverviews() as AccountOverview.OverviewsResponse;
+
+	// While the data is loading, we show the default currency tab.
+	let depositCurrencyTabs: TabPanel.Tab[] = [
+		{
+			name: wcpaySettings.accountDefaultCurrency,
+			title: getCurrencyTabTitle( wcpaySettings.accountDefaultCurrency ),
+		},
+	];
+
+	const { currencies } = overviews;
+
+	if ( ! isLoading && currencies.length !== 0 ) {
+		depositCurrencyTabs = currencies.map(
+			( overview: AccountOverview.Overview ) => ( {
+				name: overview.currency,
+				title: getCurrencyTabTitle( overview.currency ),
+				currencyCode: overview.currency,
+				availableFunds: overview.available.amount,
+				pendingFunds: overview.pending.amount,
+				reserveFunds: 0, // TODO: Add reserve funds to the overview object.
```

Comment:
If you agree that the term "Reserved funds" is consistent with the design, I think this property name should also be changed to `reservedFunds`.